sdk/data/azcosmos/cosmos_location_cache.go

```diff
@@ -40,10 +41,21 @@ type accountRegion struct {
 	Endpoint string `json:"databaseAccountEndpoint"`
 }
 
+type userConsistencyPolicy struct {
+	DefaultConsistencyLevel string `json:"defaultConsistencyLevel"`
+}
+
 type accountProperties struct {
-	ReadRegions                  []accountRegion `json:"readableLocations"`
-	WriteRegions                 []accountRegion `json:"writableLocations"`
-	EnableMultipleWriteLocations bool            `json:"enableMultipleWriteLocations"`
+	ReadRegions                  []accountRegion       `json:"readableLocations"`
+	WriteRegions                 []accountRegion       `json:"writableLocations"`
+	EnableMultipleWriteLocations bool                  `json:"enableMultipleWriteLocations"`
+	UserConsistencyPolicy        userConsistencyPolicy `json:"userConsistencyPolicy"`
+	ContinuousBackupEnabled      bool                  `json:"continousBackupEnabled"`
+}
+
+func (accountProps accountProperties) String() string {
+	return fmt.Sprintf("Read Regions: %v\nWrite Regions: %v\nUse Multiple Write Locations: %v\nDefault Consistency Level: %v\nContinous Backup Enabled: %v",
```

Comment:
```suggestion
	return fmt.Sprintf("Read Regions: %v\nWrite Regions: %v\nMulti-region writes : %v\nDefault Consistency Level: %v",
```

Portal uses "Multi-region writes"
![image](https://github.com/Azure/azure-sdk-for-go/assets/1633401/8bac0f2d-ec57-42b3-ba5c-90ad1ca9e501)

Might as well be consistent